controlpanel/api/cluster.py

```diff
@@ -63,6 +73,18 @@ def create(self):
             f"--set=Username={self.user.slug}",
         )
 
+    def reset_home(self):
+        """
+        Reset the user's home directory.
+        """
+        helm.reset_home(
+            f"reset-user-home-{self.user.slug}",
+            f"{settings.HELM_REPO}/reset-user-home",
+            "--version=0.4.1",
```

Comment:
It may be worth adding this in a setting or even getting rid of it and always install the latest version of this `reset-user-home` helm chart. 

The latter is potentially the best option as I imagine we're interested in the latest/greatest way of resetting the user home rather than to pin it to a specific version. Plus this way we don't have to get back and update/re-release the CP if that helm chart gets updated.

IMHO of course.

Reply:
:+1: What's the flag for always using the latest version of the chart..?

Reply:
OK... no need to use the `--version` flag - if it's not there, it'll always try the latest version.

Reply:
Good catch..! :+1:

Reply:
Yep I was about to reply ☺️

Reply:
Resolved in latest commit.